### PR TITLE
setup ows and explorer utility in af-cape

### PR DESCRIPTION
**Pull request by pindge**
- setup ows and explorer utility for cape town prod

### CODE REVIEW

File: dags/infra/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## images
+images used in kubenetepodoperator
```
Comment (whatnick) [inline dags/infra/README.md:2]: kubernetespodoperator

File: dags/infra/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## connections
+connections stored in SSM with prefix `/airflow/connections`
+
+## Variables
+Variables stored in SSM with prefix `/airflow/variables`
+
+## podconfig
+kubeneteoperatorpod configurations
```
Comment (whatnick) [inline dags/infra/README.md:11]: kubernetespod

File: dags/infra/images.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+# OWS_IMAGE = "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/opendatacube/ows:1.8.2"
+# OWS_CONFIG_IMAGE = "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/geoscienceaustralia/dea-datakube-config:1.5.5"
+
+
+# EXPLORER_IMAGE = (
+#     "538673716275.dkr.ecr.ap-southeast-2.amazonaws.com/opendatacube/explorer:2.2.4"
+# )
+
+INDEXER_IMAGE = "opendatacube/datacube-index:latest"
```
Comment (whatnick) [inline dags/infra/images.py:16]: Specific image version is better than latest

File: dags/infra/images.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+OWS_IMAGE = "opendatacube/ows:1.8.2"
+EXPLORER_IMAGE = "opendatacube/explorer:2.4.0"
+OWS_CONFIG_IMAGE = "geoscienceaustralia/deafrica-config:latest"
```
Comment (whatnick) [inline dags/infra/images.py:19]: Same here ... perhaps x2 variables one with latest and one with pinned stable version.

**Review by whatnick — DISMISSED** [review]

**Review by tebadi — APPROVED** [review]
I'm not quite sure what should be achieved but the code looks good to me!

**Pull request merged**